src/main/webapp/script.js

```diff
@@ -160,6 +160,57 @@ async function processImage(image) {
   drawImageOnCanvas(blurredImage.object, outputCanvas);
 
   updateDownloadButton(blurredImage);
+
+  outputCanvas.onclick = (event) => {
+    tryToUnblurArea(event, image);
+  };
+}
+
+/**
+ * Function to handle click on output canvas
+ * and toggle blurness of rects to blur
+ * if any were clicked.
+ * @param {MouseEvent} event
+ * @param {ImageObject} image
+ */
+function tryToUnblurArea(event, image) {
```

Comment:
Why is this called "try"? Is there any case when it doesn't unblur the area clicked?

Reply:
A better naming I would suggest is `toggleAreaBlur`

Reply:
Thanks! That's surely better :)